code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -153,6 +153,9 @@
 		"l_foot" = list("path" = /obj/item/organ/external/foot),
 		"r_foot" = list("path" = /obj/item/organ/external/foot/right))
 
+	// Mutant pieces
+	var/obj/item/organ/internal/ears/mutantears = /obj/item/organ/internal/ears
```

Comment:
Would it now make sense to set up Tajaran & Vulpkanin mutantears and get 'em off the head sprites? This also means we may be able to have different selectable ears (e.g. for vulp: big (normal), slim, floppy, torn, pierced, etc).
And more mechanics based on hearing tied to organs.

:thinking:
# Fuck yes

Reply:
Yes, I deliberately introduced a way for internal organs to render on the mob for cat ears, I've left that in. We can definitely tie vulp and taj ears to rendering on that instead of being hardcoded to the head sprites.

Reply:
Fantastic, thanks for clearing that up. I'm very excited